variables.yml

```diff
@@ -107,7 +107,7 @@ networks:
     min_gas_price: 1
     block_time: 12
     gas_block: 15M
-    gas_tx: 13M
+    gas_tx: 15M
```

Comment:
Our block size is 15M gas, but there may be some little bits of overhead that prevent a transaction from consuming the entire thing (it might be the case that 14999999 fits, but not 15000000, etc.) If you need an exact number, I can get it for you.

Reply:
@albertov19 do we need an exact number? could we just say "< 15M"